package.json

```diff
@@ -22,8 +22,14 @@
     "bcryptjs": "~1.0.4",
     "bower": "~1.3.8",
     "connect-flash": "~0.1.1",
+<<<<<<< HEAD
```

Comment:
@fidelisojeah This shouldn't be here. Also, the only change you made in this file is technically just adding `dotenv`

Reply:
I had to upgrade connect-mongo as well